build.sbt

```diff
@@ -156,6 +157,13 @@ lazy val commonSettings = Seq(
   homepage := Some(url("https://github.com/geminihlsw/explore")),
   licenses := Seq("BSD 3-Clause License" -> url("https://opensource.org/licenses/BSD-3-Clause")),
   scalacOptions += "-Ymacro-annotations",
+  scalacOptions ~= (_.filterNot(
+    Set(
+      // By necessity facades will have unused params
+      "-Wdead-code",
+      "-Wunused:params"
+    )
+  )),
```

Comment:
This is because we are including a simple facade for the react-moon library. We should move it to gpp-ui eventually.